### PR TITLE
#drop down

### PR DESCRIPTION
**Pull request by shubhenducw**

### CODE REVIEW

File: apps/www/registry/tui/example/cards/dropdown-card.tsx [hunk 1]
```diff
@@ -0,0 +1,409 @@
...
+  LogOut,
+  Mail,
+  MessageSquare,
+  Plus,
+  PlusCircle,
+  Settings,
+  User,
+  UserPlus,
+  Users,
+} from "lucide-react"
```
Comment (raghavcakewalk) [inline apps/www/registry/tui/example/cards/dropdown-card.tsx:18]: Please organise imports if not used

**Review by raghavcakewalk — CHANGES_REQUESTED** [review]
changes

File: apps/www/registry/tui/example/cards/dropdown-card.tsx [hunk 2]
```diff
@@ -151,6 +156,7 @@ const dropDownData: DropdownItem[] = [
 ]
 
 export function DropdownMenuCard() {
+  const [checked, setChecked] = React.useState(false);
```
Comment (raghavcakewalk) [inline apps/www/registry/tui/example/cards/dropdown-card.tsx:159]:
```suggestion
  const [checked, setChecked] = React.useState<boolean>(false);
```

File: apps/www/registry/tui/example/cards/dropdown-card.tsx [hunk 3]
```diff
@@ -161,7 +167,107 @@ export function DropdownMenuCard() {
...
+          <DropdownMenuTrigger asChild>
+            <Button variant="outline">Open</Button>
+          </DropdownMenuTrigger>
+          <DropdownMenuContent className="w-56" align="end">
+            <DropdownMenuGroup>
+              {dropDownData.map((data: DropdownItem) => {
+                return (
+                  <>
+                    {
+                      data.isLabel ? <DropdownMenuLabel>{data.title}</DropdownMenuLabel>
```
Comment (raghavcakewalk) [inline apps/www/registry/tui/example/cards/dropdown-card.tsx:232]:
```suggestion
                      data?.isLabel ? <DropdownMenuLabel>{data?.title}</DropdownMenuLabel>
```

**Review by raghavcakewalk — CHANGES_REQUESTED** [review]
need a core component and examples should be such that they are easily copy pasted
All the elements in data should be coming from props and all this if else checks should be moved to the core component

**Comment by shubhenducw** [comment]
> need a core component and examples should be such that they are easily copy pasted All the elements in data should be coming from props and all this if else checks should be moved to the core component

Core components has all the required props

**Review by sohebcakewalk — APPROVED** [review]

**Review by raghavcakewalk — APPROVED** [review]

**Review by raghavcakewalk — APPROVED** [review]

**Pull request merged**